s3-store/src/main/java/com/expedia/blobs/stores/aws/S3BlobStore.java

```diff
@@ -91,16 +93,72 @@ protected void storeInternal(Blob blob) {
             final Map<String, String> objectMetadata = s3Object.getObjectMetadata().getUserMetadata();
             try (final InputStream is = s3Object.getObjectContent()) {
                 return Optional.of(new SimpleBlob(key,
-                                                  objectMetadata == null ? new HashMap<>(0) : objectMetadata,
-                                                  readInputStream(is)));
+                        objectMetadata == null ? new HashMap<>(0) : objectMetadata,
```

Comment:
check you IDE alignment settings?

Reply:
Not sure what do you want me to change in the settings. As my intellij is aligning the code in the above way only.

Reply:
See the change above - lines have moved. It is probably Java code style in your IDE

Reply:
Not able to figure out the setting that is different in our IDEs.
![image](https://user-images.githubusercontent.com/10944151/58899890-9aa6cc00-871b-11e9-9b7c-fc0a68bd2386.png)

Reply:
I will fix the alignment